README.md

```diff
@@ -1 +1,35 @@
-# sdcore-upf-snap
\ No newline at end of file
+# SD-Core User Plane Function (UPF) Snap
+
+A snap for the SD-Core User Plane Function
+
+## Usage
+
+### Install
+
+```bash
+sudo snap install sdcore-upf
```

Comment:
Actually right now it will be devmode because we need to make a request to have it classic. I made the necessary change here.

Before making the request for classic we should investigate strict confinement but that will be for a future task.

Reply:
Accidentally resolved this convo and added `--devmode` to the snapcraft installation cmd. If we'll be using `devmode`, we need to add `--devmode` here, don't we?

Reply:
Yes, done!